experimental/regression_suite/tests/pregenerated/test_llama2.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers import *
+
+###############################################################################
+# Fixtures
+###############################################################################
+
+COMMON_FLAGS = [
+    "--iree-input-type=none",
+    "--iree-stream-resource-index-bits=64",
+    "--iree-vm-target-index-bits=64",
+    "--iree-opt-const-expr-hoisting=false",
+    "--iree-stream-resource-max-allocation-size=3221225472",
+]
+
+llama2_7b_f16qi4_source = fetch_source_fixture(
+    "https://storage.googleapis.com/shark_tank/stella/llama2_7b_int4_stripped.mlir",
+    group="llama2_7b_f16qi4",
+)
+
+
+@pytest.fixture
+def llama2_7b_f16qi4_rdna3_vulkan_vmfb(llama2_7b_f16qi4_source):
+    return iree_compile(
```

Comment:
I guess in the future it is possible to separate the compilation and test/benchmark stages in CI? So we don't need to use expensive GPU machines to compile modules

Reply:
For the kind of testing we are doing, it is easily not worth it: some of these regression tests are compiling 35GiB models and we don't want to be persisting that across a network when almost all of the time is already taken by moving the weights around.

I was going to add some modes to run this in where it can prep an artifact dir but not actually run. Then you could transport that and run for real if desired. But not in this round.